src/components/elements/carousel/carousel.navigation-container.tsx

```diff
@@ -0,0 +1,67 @@
+import styled, { css } from 'styled-components'
+import React from 'react'
+import useCarousel from './use-carousel'
+import { TCarouselMode, TNavPlacement, TOffset } from './carousel.types'
+import { getOffsetValue } from './utils'
+
+type TNavigationWrapperProps = {
+    placement: TNavPlacement
+    mode: TCarouselMode
+    offset: TOffset
+}
+
+const NavigationWrapper = styled.div<TNavigationWrapperProps>`
+    display: flex;
+    position: absolute;
+    justify-content: center;
+    align-items: center;
+    gap: 8px;
+    ${({ placement, mode }) => {
+        if (mode === 'vertical') {
+            if (placement === 'start') {
+                return css<{ offset: TOffset }>`
+                    height: 100%;
+                    left: ${({ offset }) => getOffsetValue(offset)};
+                    top: 0;
+                    width: 8px;
+                    flex-direction: column;
+                `
+            }
+            return css<{ offset: TOffset }>`
+                height: 100%;
+                right: ${({ offset }) => getOffsetValue(offset)};
+                top: 0;
+                width: 8px;
+                flex-direction: column;
+            `
+        } else {
+            if (placement === 'start') {
+                return css<{ offset: TOffset }>`
+                    width: 100%;
+                    top: ${({ offset }) => getOffsetValue(offset)};
+                    height: 8px;
+                    flex-direction: row;
+                `
+            }
+            return css<{ offset: TOffset }>`
+                width: 100%;
+                bottom: ${({ offset }) => getOffsetValue(offset)};
+                height: 8px;
+                flex-direction: row;
+            `
+        }
+    }}
```

Comment:
```suggestion
    ${({ placement, mode }) => {
        let is_vertical = mode === 'vertical'
        let is_placement_start = placement === 'start'
        let vertical_direction = is_placement_start ? 'left' : 'right'
        let horizontal_direction = is_placement_start ? 'top' : 'bottom'
        let height = is_vertical ? '100%' : '8px'
        let width = is_vertical ? '8px' : '100%'
        let flex_direction = is_vertical ? 'column' : 'row'

        return css<{ offset: TOffset }>`
            ${is_vertical ? vertical_direction : horizontal_direction}: ${({ offset }) => getOffsetValue(offset)};
            ${is_vertical ? 'top: 0' : ''};
            flex-direction: ${flex_direction};
            height: ${height};
            width: ${width};
        `
    }}
```

Reply:
cool, thanks thanks :pray: 
applied the suggested change.